ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -73,12 +73,13 @@ StackLayout {
             const contactAddr = chatsModel.suggestionList.rowData(i, "address");
             if(idMap[contactAddr]) continue;
             const contactIndex = profileModel.contacts.list.getContactIndexByPubkey(chatsModel.suggestionList.rowData(i, "address"));
-
+            const isContact = profileModel.contacts.list.rowData(contactIndex, "isContact");
+            const thumbnail = profileModel.contacts.list.rowData(contactIndex, "thumbnailImage")
             suggestionsObj.push({
                                     alias: chatsModel.suggestionList.rowData(i, "alias"),
                                     ensName: chatsModel.suggestionList.rowData(i, "ensName"),
                                     address: contactAddr,
-                                    identicon: profileModel.contacts.list.rowData(contactIndex, "thumbnailImage"),
+                                    identicon: isContact === "true" && thumbnail ? thumbnail : chatsModel.suggestionList.rowData(i, "identicon"),
```

Comment:
We can just use the `getProfileImage` function directly. It handles the contact check and that fallback https://github.com/status-im/status-desktop/blob/master/ui/app/AppMain.qml#L27

Reply:
I definitely did not see that function, thank you! Updated to use `getProfileImage`, though I did need to add a condition to handle the case when the pubkey passed in was not a contact to use the identicon instead.